### PR TITLE
HP-727: HDS Update

### PR DESCRIPTION
**Pull request by NikoHelle**
Updated HDS to latest (1.1.0.) Small changes except with new Dropdowns and Dialog.

Body-element uses base HDS font-size and line-height. All "px" font sizes are replaced with HDS font sizes.

A lot of files are changed because all CSS classnames were renamed from 'className' to 'class-name'. This is the  coding standard defined in dev.hel.fi

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
     "cancel": "Cancel",
     "city": "City",
     "country": "Country",
-    "delete": "Delete",
+    "delete": "profileForm┊delete",
```
Comment (akikoskinen) [inline src/i18n/en.json]: What is this change? Is this translation actually used any where?

**Review by akikoskinen — COMMENTED** [review]
Looks nice.

I did notice that the dialog that is shown when the user is about to delete the profile, has focus on the title and it draws a thick border around it.
modal-title-focus

Another observation, probably not related to this PR however: no matter what UI language the user selects, the header always says "Profile", and page title (the one within the <title></title> tags in HTML) is always "Profiili".

**Comment by NikoHelle** [comment]
> Looks nice.
> 
> I did notice that the dialog that is shown when the user is about to delete the profile, has focus on the title and it draws a thick border around it.
> modal-title-focus
> 

That happens every time a dialog is opened. Also in  [HDS examples](https://hds.hel.fi/storybook/react/?path=/story/components-dialog--default) Focus is set for better accessibility. The focus border could be prettier, IMHO :). The heading is created by the HDS component, so I'm not overriding it at the moment.

> Another observation, probably not related to this PR however: no matter what UI language the user selects, the header always says "Profile", and page title (the one within the <title></title> tags in HTML) is always "Profiili".

Good point. I've added translations in a new commit.

File: src/i18n/en.json (hunk 1 above)
Comment (NikoHelle) [reply src/i18n/en.json]: That's odd. I reverted that change.

**Comment by akikoskinen** [comment]
> > I did notice that the dialog that is shown when the user is about to delete the profile, has focus on the title and it draws a thick border around it.
> 
> That happens every time a dialog is opened. Also in [HDS examples](https://hds.hel.fi/storybook/react/?path=/story/components-dialog--default) Focus is set for better accessibility. The focus border could be prettier, IMHO :). The heading is created by the HDS component, so I'm not overriding it at the moment.

Yep, no point trying to circumvent HDS features. But could it be communicated to HDS devs that this part could maybe use some improvement?

**Comment by NikoHelle** [comment]
> > > I did notice that the dialog that is shown when the user is about to delete the profile, has focus on the title and it draws a thick border around it.
> > 
> > 
> > That happens every time a dialog is opened. Also in [HDS examples](https://hds.hel.fi/storybook/react/?path=/story/components-dialog--default) Focus is set for better accessibility. The focus border could be prettier, IMHO :). The heading is created by the HDS component, so I'm not overriding it at the moment.
> 
> Yep, no point trying to circumvent HDS features. But could it be communicated to HDS devs that this part could maybe use some improvement?

Has been done: [https://helsinkicity.slack.com/archives/CHCV3KTHA/p1623933531040900](https://helsinkicity.slack.com/archives/CHCV3KTHA/p1623933531040900)

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**